bpfctl/src/main.rs

```diff
@@ -165,6 +165,31 @@ enum LoadCommands {
         #[clap(short, long, verbatim_doc_comment)]
         tracepoint: String,
     },
+    /// Install an eBPF uprobe
+    Uprobe {
+        /// Optional: function to attach the uprobe to.
+        #[clap(short, long)]
+        fn_name: Option<String>,
+
+        /// Optional: offset added to the address of the target function
+        /// (or beginning of target if no function is identified)
+        #[clap(short, long)]
+        offset: Option<u64>,
+
+        /// Required: absolute path to a binary or library, or a library name
```

Comment:
```suggestion
        /// Required: absolute path to a binary, library, or library name
```

Reply:
"absolute path" only applies to "binary" and "library", and not to "library name".  So I think the way I wrote it is technically correct.  However, it may not be clear.  I'll change it to the following:
"library name or the absolute path to a binary or library"